src/coreclr/inc/corinfo.h

```diff
@@ -2265,6 +2265,13 @@ class ICorStaticInfo
             int                         bufferSize  /* IN  */
             ) = 0;
 
+    // Calls ToString() for given pinned/frozen object handle
```

Comment:
May be worth to document that the buffer is not null terminated.

Reply:
Done